Examples/Python/src/MagneticField.cpp

```diff
@@ -45,7 +45,9 @@ void addMagneticField(Context& ctx) {
 
   py::class_<Acts::InterpolatedMagneticField,
              std::shared_ptr<Acts::InterpolatedMagneticField>>(
-      m, "InterpolatedMagneticField");
+      m, "InterpolatedMagneticField")
+      .def("getFieldUnchecked",
+           &Acts::InterpolatedMagneticField::getFieldUnchecked);
```

Comment:
Can you explain what "Unchecked" means here?

Reply:
I removed it and added python bindings for getField instead (it's used for the python tests).